core/shared/src/main/scala/scalaz/zio/RTS.scala

```diff
@@ -0,0 +1,20 @@
+package scalaz.zio
+
+import scalaz.zio.internal.Env
+import scalaz.zio.ExitResult.Cause
+
+trait RTS {
```

Comment:
So what's the added value of RTS currently? It's only delegating to `Env`. Or would it be evolved in latter PRs?

Reply:
It'll probably be deleted. For now it's just a mixin trait to easily get `unsafeRun*` for benchmarking and tests.